packages/types/jest.config.js

```diff
@@ -0,0 +1,20 @@
+module.exports = {
+    "preset": "ts-jest",
+    "transform": {
+        "^.+\\.(j|t)s?$": "ts-jest"
+    },
+    "testMatch": [
+        "<rootDir>/lib/__tests__/*.test.ts"
+    ],
+    "collectCoverageFrom": [
+        "<rootDir>/lib/index.ts"
```

Comment:
tests will be added later?

Reply:
No, there is no point to test types. I'll drop this.